subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -173,10 +173,9 @@ uint8_t ll_sync_create_cancel(void **rx)
 	struct ll_scan_set *scan_coded;
 	memq_link_t *link_sync_estab;
 	memq_link_t *link_sync_lost;
-	struct node_rx_pdu *node_rx;
```

Comment:
The compiler is likely smart enough that removing this auto variable does not save anything but only makes the code a little harder to read.